### PR TITLE
Checkpoint 2: Components

### PR DESCRIPTION
**Pull request by CRClothier**
In this milestone:

- the placeholder 'create react app' content was removed .
- the JSX for the calculator was added as a separate component.
- the calculator was styled following the reference image.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,10 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
     </div>
   );
```
Comment (bimbolabuari) [inline src/App.js]: As part of the project requirement, Kindly refactor this to a class-based component 🙂 

image

File: src/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      <button type="button">2</button>
+      <button type="button">3</button>
+      <button type="button" className="orange">+</button>
+      <button type="button" className="zero">0</button>
+      <button type="button">.</button>
+      <button type="button" className="orange">=</button>
+    </div>
+  );
+}
+
```
Comment (bimbolabuari) [inline src/Calculator.js]: As required, Kindly refactor this to use a class-based component instead of a function component

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @CRClothier, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Delete all the boilerplate from CRA (text, images, styles).
- Create a `calculator.js` file
- import calculator.js and use it in your main component (App.js).

## Required Changes ♻️
- [ ] As required, Kindly create a directory called `components` in the src folder, Inside components, add your Calculator.js file.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (CRClothier) [reply src/App.js]: HI @bimbolabuari ,

Thank you kindly for your review and feedback, I just have one question:

 I used class based components initially, but it caused a linter error saying to use functions instead.  Is it ok if I ignore these linter errors?

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi @CRClothier, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**